app/controllers/coronavirus/pages_controller.rb

```diff
@@ -26,13 +26,13 @@ def discard
       else
         message = { alert: draft_updater.errors.to_sentence }
       end
-      redirect_to coronavirus_page_path(slug), message
+      redirect_to coronavirus_page_path(page.slug), message
     end
 
   private
 
     def page
-      @page ||= Pages::ModelBuilder.call(slug)
+      @page ||= Page.find_by(slug: slug)
```

Comment:
I think the model builder served a purpose that would now be lost of how to initially create a page if one does not exist in the database - effectively acting as a lazy load mechanism. Is there an alternative process as to how that is supposed to happen?

By the way, this should be `find_by!` so that this will return a 404 if this fails.

Reply:
The initialisation still happens when the index page is loaded. That still uses `ModelBuilder` and hasn't been removed. It's called by the `initialise_pages` before action.

Reply:
Oh cool - sorry I didn't realise that.